cocos/core/utils/misc.ts

```diff
@@ -26,15 +26,16 @@
 /* eslint-disable no-new-func */
 
 import { DEBUG, DEV } from 'internal:constants';
-import { setTimeoutRAF } from '@pal/utils';
+import { isDescendantElementOf, isDomNode as _isDomNode, setTimeoutRAF } from '@pal/utils';
 import { cclegacy } from '@base/global';
 import { warnID } from '@base/debug';
 import { js } from '@base/utils';
 import { isPlainEmptyObj } from '@base/utils/internal';
-import { macro } from '../platform/macro';
-
-const { getClassName, getset, isEmptyObject } = js;
+import { toRadian, toDegree } from '../math';
 
+/**
+ * @deprecated since v3.9.0
+ */
 export const BUILTIN_CLASSID_RE = /^(?:cc|dragonBones|sp|ccsg)\..+/;
```

Comment:
Any replacement?

Reply:
no, this should be an engine internal interface, should be removed in the future

Reply:
it is only used for `deserialize-dynamic.ts`,

In the past, we put all undesigned interfaces in the `misc` module, and also exposed all the undesigned interfaces with the `misc` module, it's weired.

Reply:
Got it.